.pre-commit-config.yaml

```diff
@@ -0,0 +1,40 @@
+---
+repos:
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v4.4.0
+    hooks:
+      - id: end-of-file-fixer
+      - id: trailing-whitespace
+      - id: mixed-line-ending
+      - id: check-byte-order-marker
+      - id: check-executables-have-shebangs
+      - id: check-merge-conflict
+      - id: debug-statements
+      - id: check-json
+      - id: check-yaml
+        files: .*\.(yaml|yml)$
+        args: ["--unsafe"]
+  - repo: https://github.com/PyCQA/flake8.git
+    rev: 3.8.3
+    hooks:
+      - id: flake8
+  - repo: https://github.com/adrienverge/yamllint.git
+    rev: v1.29.0
+    hooks:
+      - id: yamllint
+        files: \.(yaml|yml)$
+        types: [file, yaml]
+        entry: yamllint --strict -f parsable
+  - repo: https://github.com/ansible/ansible-lint.git
+    rev: v6.11.0
+    hooks:
+      - id: ansible-lint
+        always_run: true
+        pass_filenames: false
+        verbose: true
+        entry: bash -c "ANSIBLE_LIBRARY=plugins/modules ansible-lint --force-color -p -v"
```

Comment:
I'm not completely sure why is the path being set here. The repo doesn't have a plugins directory so it wouldn't point to anything.

Reply:
True. Removed 'ANSIBLE_LIBRARY'